content/authoring/kata/index.md

```diff
@@ -1,12 +1,30 @@
 ---
-kind: tutorial
 sidebar: authoring
 next: /authoring/
 prev: /authoring/translation/
 ---
 
-# How to create a kata
+# Kata
 
-_This page is a stub._
+_TBD: improve this stub_
 
+## Requirements
 
+To create a new kata, a user needs to earn the ["Create Kata" privilege][reference-privileges]. After reaching the required amount of Honor, the privilege is granted and the user can select the **"New Kata"** option from their profile menu.
+
+
+## Kata Editor
+
+Kata are created by filling the 
```

Comment:
??? (incomplete sentence)

Reply:
This page is still TDB and out of scope of this PR, but I've at least completed the sentence :)